setup.py

```diff
@@ -1,7 +1,6 @@
-from codecs import open
 from os import path
 
-from setuptools import Extension, find_packages, setup
+from setuptools import Extension, setup
 
 """
 Release instruction:
```

Comment:
In this docstring, change:

If you ever change the
`requirements[_dev].txt`, also update the

to:

If you ever change the
`pyproject.toml`, also update the

and delete the requirements.txt and requirements_dev.txt files.

Reply:
In docstring, change:

Change __version__ in setup.py to new version name.

to:

Change version in pyproject.toml to new version name.

Reply:
Done ✅